src/templates/drawerDeprecated.js

```diff
@@ -147,13 +147,12 @@ class DrawerDeprecatedHeader extends Component {
 
     _onHeaderUpdate(node) {
         const headerEl = ReactDOM.findDOMNode(this);
-
         if (headerEl) {
             const paddingTop = headerEl.offsetHeight;
-
-            if (paddingTop !== this._drawerContainerInnerPaddingTop) {
+            let drawerInnerContainer = headerEl.closest('.drawer-container-inner'); 
```

Comment:
I suppose that `drawer-container-inner` is a standard class for the container? Is this a safe assumption?

Reply:
Yes, it is an standard, the container, element with class it's created inside the render method, I suggest to move the class name to a constant in a separate task.

Reply:
Yes, that is a safe assumption. It looks that `drawerInnerContainer` is not being redefined and should be a `const`.

Reply:
Done! Commit pushed :) thanks.